### PR TITLE
FixBlogLinks: updates to 10 articles

### PR DESCRIPTION
**Pull request by craigcaseyMSFT**
Updated or removed blog links in 10 articles per mseng User Story 1443043.

Contributes to #9807

### CODE REVIEW

**Comment by craigcaseyMSFT** [comment]
@mairaw  - Hi, Maira. Please review these first 10 articles and let me know if anything should be changed before we can merge. Thanks for your help!

File: docs/core/docker/building-net-docker-images.md [hunk 1]
```diff
@@ -270,7 +270,6 @@ Here are some next steps you can take:
 * [Working with Visual Studio Docker Tools](https://docs.microsoft.com/aspnet/core/publishing/visual-studio-tools-for-docker)
 * [Deploying Docker Images from the Azure Container Registry to Azure Container Instances](https://blogs.msdn.microsoft.com/stevelasker/2017/07/28/deploying-docker-images-from-the-azure-container-registry-to-azure-container-instances/)
 * [Debugging with Visual Studio Code](https://code.visualstudio.com/docs/nodejs/debugging-recipes#_nodejs-typescript-docker-container) 
-* [Getting hands on with Visual Studio for Mac, containers, and serverless code in the cloud](https://blogs.msdn.microsoft.com/visualstudio/2017/08/31/hands-on-with-visual-studio-for-mac-containers-serverless-code-in-the-cloud/#comments)
```
Comment (mairaw) [inline docs/core/docker/building-net-docker-images.md]: hmm, this one is interesting. Let me follow up with the PM driving the migration to see if this is expected.

File: docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md [hunk 2]
```diff
@@ -119,10 +119,10 @@ You can specify additional configuration settings in the Dockerfile, depending o
   [*https://docs.docker.com/engine/tutorials/dockerimages/*](https://docs.docker.com/engine/tutorials/dockerimages/)
 
 - **Staying up-to-date with .NET Container Images** \
-  [*https://blogs.msdn.microsoft.com/dotnet/2018/06/18/staying-up-to-date-with-net-container-images/*](https://blogs.msdn.microsoft.com/dotnet/2018/06/18/staying-up-to-date-with-net-container-images/)
+  [*https://devblogs.microsoft.com/dotnet/staying-up-to-date-with-net-container-images/*](https://devblogs.microsoft.com/dotnet/staying-up-to-date-with-net-container-images/)
```
Comment (mairaw) [inline docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md]: Let's change this link format to this:

```suggestion
  *<https://devblogs.microsoft.com/dotnet/staying-up-to-date-with-net-container-images/>*
```

File: docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md [hunk 3]
```diff
@@ -119,10 +119,10 @@ You can specify additional configuration settings in the Dockerfile, depending o
...
 
 - **Using .NET and Docker Together - DockerCon 2018 Update** \
-  [*https://blogs.msdn.microsoft.com/dotnet/2018/06/13/using-net-and-docker-together-dockercon-2018-update/*](https://blogs.msdn.microsoft.com/dotnet/2018/06/13/using-net-and-docker-together-dockercon-2018-update/)
+  [*https://devblogs.microsoft.com/dotnet/using-net-and-docker-together-dockercon-2018-update/*](https://devblogs.microsoft.com/dotnet/using-net-and-docker-together-dockercon-2018-update/)
```
Comment (mairaw) [inline docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md]:
```suggestion
  *<https://devblogs.microsoft.com/dotnet/using-net-and-docker-together-dockercon-2018-update/>*
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @craigcaseyMSFT! Changes look good. I left a couple of suggestions to simplify some links and also I'd revert the change to remove that 404 link until I can get confirmation that's by design.

**Comment by mairaw** [comment]
I also added to the description a link to our GitHub issue tracking this work. Can you do the same when you open the new PRs?

**Comment by mairaw** [comment]
@craigcaseyMSFT can you also undo the change done for building-net-docker-images.md as requested? Then we can merge this.

**Comment by craigcaseyMSFT** [comment]
Change undone for building-net-docker-images.md as requested. Ready to merge.

#sign-off

**Review by mairaw — APPROVED** [review]
Thank you @craigcaseyMSFT!

**Pull request merged**